master/internal/api_workspace.go

```diff
@@ -519,12 +558,31 @@ func (a *apiServer) deleteWorkspaceNamespaceBinding(ctx context.Context,
 			if err != nil {
 				return err
 			}
+			deletedClusters := []string{}
 			for _, v := range deletedBindings {
+				deletedClusters = append(deletedClusters, v.ClusterName)
 				err = a.m.rm.RemoveEmptyNamespace(v.Namespace, v.ClusterName)
 				if err != nil {
 					return err
 				}
 			}
+			if len(deletedClusters) < len(clusterNames) {
+				switch {
+				case len(clusterNames) == 1 && (clusterNames[0] != ""):
+					return fmt.Errorf("tried to delete default binding for cluster %v", clusterNames[0])
+				case len(clusterNames) == 1:
+					return fmt.Errorf("tried to delete default binding for cluster")
```

Comment:
Can we do 
```suggestion
				case len(clusterNames) == 1:
					return fmt.Errorf("tried to delete default binding for cluster %v", clusterNames[0])
```
since when `clusterNames[0]` is empty, we get the same resulting error message

Reply:
done